backend/src/org/commcare/session/CommCareSession.java

```diff
@@ -356,14 +358,14 @@ public Suite getCurrentSuite() {
         return null;
     }
 
-    public void stepBack() {
```

Comment:
Yeah, I hope to revisit this and clean it up. I tried to go that route but they are pretty deeply interdependent.